src/promptflow/promptflow/executor/_errors.py

```diff
@@ -165,6 +165,10 @@ class NodeOutputNotFound(UserErrorException):
     pass
 
 
+class SingleNodeValidationError(UserErrorException):
+    pass
```

Comment:
add one line summary?

Reply:
this name is clear and this is non-public